detection_rules/beats.py

```diff
@@ -64,16 +64,35 @@ def _decompress_and_save_schema(url, release_name):
         f.write(compressed)
 
 
+def download_beats_schema(version: str):
+    """Download a beats schema by version"""
+    url = 'https://api.github.com/repos/elastic/beats/releases'
+    releases = requests.get(url)
+
+    version = f'v{version.lstrip("v")}'
+    beats_release = None
+    for release in releases.json():
+        if release['tag_name'] == version:
+            beats_release = release
+            break
+
+    if not beats_release:
+        print(f'beats release {version} not found!')
+        return
+
+    beats_url = beats_release['zipball_url']
+    name = beats_release['tag_name']
+
+    _decompress_and_save_schema(beats_url, name)
+
+
 def download_latest_beats_schema():
     """Download additional schemas from beats releases."""
     url = 'https://api.github.com/repos/elastic/beats/releases'
     releases = requests.get(url)
 
     latest_release = max(releases.json(), key=lambda release: Version(release["tag_name"].lstrip("v")))
```

Comment:
it will iterate releases twice now, but I think it should be fine

Reply:
sounds fine. it's not like we download the full zip every time